app/services/tag_moderators/add.rb

```diff
@@ -1,18 +1,21 @@
 module TagModerators
   class Add
-    def self.call(user_ids, tag_ids)
-      new(user_ids, tag_ids).call
+    Result = Struct.new(:success?, :errors, keyword_init: true)
+
+    def self.call(user_id, tag_id)
+      new(user_id, tag_id).call
     end
 
-    def initialize(user_ids, tag_ids)
-      @user_ids = user_ids
-      @tag_ids = tag_ids
+    def initialize(user_id, tag_id)
```

Comment:
Simplified the class: we are not using adding multiple tag mods at once, also the code was a bit counterintuitive.